config/geoip.php

```diff
@@ -0,0 +1,165 @@
+            'locales' => ['en'],
+        ],
+
+        'maxmind_api' => [
+            'class' => \Torann\GeoIP\Services\MaxMindWebService::class,
+            'user_id' => env('MAXMIND_USER_ID'),
+            'license_key' => env('MAXMIND_LICENSE_KEY'),
+            'locales' => ['en'],
+        ],
+
+        'ipapi' => [
+            'class' => \Torann\GeoIP\Services\IPApi::class,
+            'secure' => true,
+            'key' => env('IPAPI_KEY'),
+            'continent_path' => storage_path('app/continents.json'),
+            'lang' => 'en',
+        ],
+
+        'ipgeolocation' => [
+            'class' => \Torann\GeoIP\Services\IPGeoLocation::class,
+            'secure' => true,
+            'key' => env('IPGEOLOCATION_KEY'),
+            'continent_path' => storage_path('app/continents.json'),
+            'lang' => 'en',
+        ],
+
+        'ipdata' => [
+            'class'  => \Torann\GeoIP\Services\IPData::class,
+            'key'    => env('IPDATA_API_KEY'),
+            'secure' => true,
+        ],
+
+        'ipfinder' => [
+            'class'  => \Torann\GeoIP\Services\IPFinder::class,
+            'key'    => env('IPFINDER_API_KEY'),
+            'secure' => true,
+            'locales' => ['en'],
+        ],
+
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Cache Driver
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify the type of caching that should be used
+    | by the package.
+    |
+    | Options:
+    |
+    |  all  - All location are cached
+    |  some - Cache only the requesting user
+    |  none - Disable cached
+    |
+    */
+
+    'cache' => 'all',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Tags
+    |--------------------------------------------------------------------------
+    |
+    | Cache tags are not supported when using the file or database cache
+    | drivers in Laravel. This is done so that only locations can be cleared.
+    |
+    */
+
+    'cache_tags' => null,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Expiration
+    |--------------------------------------------------------------------------
+    |
+    | Define how long cached location are valid.
+    |
+    */
+
+    'cache_expires' => 30,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Location
+    |--------------------------------------------------------------------------
+    |
+    | Return when a location is not found.
+    |
+    */
+
+    'default_location' => [
+        'ip' => '127.0.0.0',
+        'iso_code' => 'US',
+        'country' => 'United States',
+        'city' => 'New Haven',
+        'state' => 'CT',
```

Comment:
Can we change this to default to Atlanta... just because?

Reply:
Done :)